packages/primitives/list/meta.tsx

```diff
@@ -0,0 +1,17 @@
+import type { TComponentConfig } from 'autoprops'
```

Comment:
it's not working for me on native :/ 
<img width="446" alt="Screenshot 2021-01-18 at 12 06 39" src="https://user-images.githubusercontent.com/7669672/104907596-a04f3500-5985-11eb-9898-156b2df07531.png">

Reply:
just tried again now and it works for me 🤔

Reply:
it was something on my end, you can ignore this

Reply:
Works well on android for me as well. @moriaam maybe try if things are working for you on master?
![image](https://user-images.githubusercontent.com/13185981/104916732-c0d1bc00-5992-11eb-80dd-752cb7d60529.png)

Reply:
@moriaam Sorry, didn't refresh the page before posting this, I didn't see you already answered 😓 I'm happy all is fine :)